android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.kt

```diff
@@ -354,19 +354,15 @@ abstract class ReactNativeActivity :
       KernelConstants.IS_HEADLESS_KEY to false
     )
 
-    val instanceManagerBuilderProperties = InstanceManagerBuilderProperties().apply {
-      this.application = this@ReactNativeActivity.application
-      this.jsBundlePath = this@ReactNativeActivity.jsBundlePath
-      this.experienceProperties = experienceProperties
-      this.expoPackages = extraExpoPackages
-      this.exponentPackageDelegate = delegate.exponentPackageDelegate
-      this.manifest = this@ReactNativeActivity.manifest
-      this.singletonModules = ExponentPackage.getOrCreateSingletonModules(
-        this@ReactNativeActivity.applicationContext,
-        this@ReactNativeActivity.manifest,
-        extraExpoPackages
-      )
-    }
+    val instanceManagerBuilderProperties = InstanceManagerBuilderProperties(
+      application = application,
+      jsBundlePath = jsBundlePath,
+      experienceProperties = experienceProperties,
+      expoPackages = extraExpoPackages,
+      exponentPackageDelegate = delegate.exponentPackageDelegate,
+      manifest = manifest!!,
```

Comment:
when it's such a long list it can help to use named arguments to ensure that ordering isn't mixed up. while all the types here differ so the risk is lower, it still seemed cleaner.